src/graphnode.ts

```diff
@@ -13,6 +13,7 @@ import { Observable, Observer } from 'rxjs'
 import { catchError, filter, first, map } from 'rxjs/operators'
```

Comment:
Let's move these types of files into a `utils` folder within `src`

Reply:
Moving rxjs and other packages's imports into a utils file and then re exporting them?

Reply:
whoops, no I meant the whole graphnode file :P I went ahead and did this